src/sources/splunk_hec.rs

```diff
@@ -216,19 +219,14 @@ impl SplunkSource {
             .and(self.authorization())
             .and(splunk_channel)
             .and(warp::addr::remote())
+            .and(warp::header::optional::<String>("X-Forwarded-For"))
             .and(self.gzip())
             .and(warp::body::bytes())
             .and_then(
-                move |_,
-                      _,
-                      channel: String,
-                      remote: Option<SocketAddr>,
-                      gzip: bool,
-                      body: Bytes| {
+                move |_, _, channel: String, remote: Option<SocketAddr>, xff: Option<String>, gzip: bool, body: Bytes| {
```

Comment:
I totally just noticed that I flattened this out to a line instead of the vertical notation. Would you all prefer I revert this to the original style?

Reply:
Whatever `cargo fmt` says.